backends/cuda/runtime/cuda_backend.cpp

```diff
@@ -83,7 +83,36 @@ class ET_EXPERIMENTAL CudaBackend final
       return false;
     }
     std::lock_guard<std::mutex> guard(skip_copy_method_mutex_);
-    return method_name == skip_copy_method_;
+    // Support comma-separated list of method names
+    if (skip_copy_method_.empty()) {
```

Comment:
hmm can we make the `skip_copy_method_` as an array? Maybe we can avoid the fancy comparsion here for supporting multiple method names